rust/Cargo.toml

```diff
@@ -77,14 +77,15 @@ glibc_version = { path = "../glibc_version", version = "0.1" }
 
 [features]
 default = ["arrow", "parquet"]
```

Comment:
should we also add aws-profile in the default feature list? profile based auth is a very common practice in production environment.

Reply:
maybe add it as default to s3* features, since those are the only ones where it is useful?

Reply:
Agreed!